test/functionalTest/cases/0000_large_requests/notification_different_sizes.test

```diff
@@ -240,9 +240,9 @@ Date: REGEX(.*)
 ** Too large errors:
 1
 ** Notification sizes:
-Sending message 1 to HTTP server: sending message of REGEX((19485|19490|19486|19491)) bytes to HTTP server
-Sending message 2 to HTTP server: sending message of REGEX((21485|21490|21486|21491)) bytes to HTTP server
-Sending message 3 to HTTP server: sending message of REGEX((8100487|8100492|8100488|8100493)) bytes to HTTP server
+Sending message 1 to HTTP server: sending message of 19500 bytes to HTTP server
+Sending message 2 to HTTP server: sending message of 21500 bytes to HTTP server
+Sending message 3 to HTTP server: sending message of 8100502 bytes to HTTP server
```

Comment:
Please don't collapse all the 4 cases into just one, taking into account the note bellow:

```
# NOTE: the different sizes above is to cover for different sizes in the version string
#
#       d.d.d         (+0)
#       d.d.d-next    (+5)
#       d.dd.d        (+1)
#       d.dd.d-next   (+6)
```